exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/InternalBatch.java

```diff
@@ -88,4 +91,7 @@ public void clear() {
     return container.getValueAccessorById(clazz, fieldIds);
   }
 
+  public FragmentContext getContext() {
+    return incoming.getContext();
```

Comment:
Done. I use the incoming to get/store the context - not needed to pass it via the constructor.